src/EventStore.Licensing/Keygen/KeygenLifecycleService.cs

```diff
@@ -259,7 +249,16 @@ static bool TryGetParsedResponse<T>(
 }
 
 static class ErrorExtensions {
-	public static LicenseInfo.Conclusive ToConclusiveInfo(this KeygenError error) {
-		return LicenseInfo.Conclusive.FromError($"{error.Title}. {error.Detail}");
-	}
+	static readonly string[] _conclusiveErrors = [
+		"LICENSE_INVALID",
+		"LICENSE_SUSPENDED",
+		"MACHINE_CORE_LIMIT_EXCEEDED",
+		"MACHINE_LIMIT_EXCEEDED",
+		"MACHINE_PROCESS_LIMIT_EXCEEDED",
```

Comment:
Should there be a test for the `MACHINE_PROCESS_LIMIT_EXCEEDED` case?

Reply:
I decided to include it - so far we aren't using this but it gives us the option in the keygen policy if we want to and it's clear what it means